src/index.css

```diff
@@ -70,3 +70,11 @@ button {
   padding-top: 10px;
   padding-bottom: 10px;
 }
+
+.reason-for-send {
```

Comment:
@AlexeyKosinski Because the react-clamp library which I added is not compatible with jss styles. But it can be styled by passing className property - what I did.